df2gspread/utils.py

```diff
@@ -50,6 +48,13 @@ def run(cmd):
         sys.exit(process.returncode)
     return output, errors
 
+def convert_to_oauth_credentials(token_file=None):
```

Comment:
Can you also change modify `get_credentials()` function which has almost the same part of code (DRY)?
Otherwise LGTM!